src/rt/CMakeLists.txt

```diff
@@ -155,6 +155,15 @@ if(NOT DEFINED VERONA_RT_ONLY_HEADER_LIBRARY)
     set (TOP_SEED 20)
   endif ()
 
+  # Look for lldb 
+  find_program(LLDB NAMES lldb)
+
+  # For CI builds attempt to run inside LLDB to get more information
+  # for crashes.
+  if (VERONA_CI_BUILD AND (NOT LLDB STREQUAL LLDB-NOTFOUND))
```

Comment:
Running every test inside a debugger will incur in performance penalty for normal runs. Perhaps we could make this a separate flag and only check if LLDB exists and use it with that flag, regardless of where we build.

Reply:
The problem is if there is ever a crash, we want as much information immediately.  If everything is super repeatably, then it is fine.  As far as I understand this is a fixed cost per test.  To the runtime, they all run a reasonable amount inside.  I'll measure the time difference though, to see what we are paying for this.